tests/core/failover/task.yaml

```diff
@@ -21,30 +21,58 @@ environment:
     INJECT_FAILURE/emptysystemd: inject_emptysystemd_failure
     # FIXME: disabled until we find what to do!
     # fails with:  ERROR cannot replace signed kernel snap with an unasserted one
-    #INJECT_FAILURE/emptyinitrd: inject_emptyinitrd_failure
+    INJECT_FAILURE/emptyinitrd: inject_emptyinitrd_failure
     TARGET_SNAP/rclocalcrash: core
     TARGET_SNAP/emptysystemd: core
-    #TARGET_SNAP/emptyinitrd: kernel
+    TARGET_SNAP/emptyinitrd: kernel
```

Comment:
we can't keep TARGET_SNAP/emptyinitrd: core and TARGET_SNAP/emptyinitrd: kernel, it is going to use always the kernel value.
In this case we will need more variants like TARGET_SNAP/emptyinitrd_core and TARGET_SNAP/emptyinitrd_kernel